### ISSUE TITLE
Corridor rounded corners

### ISSUE THREAD

**Issue by strbam** [AnalyticalGraphicsInc/cesium#8609]
Rounded corners are cropped based on angle. This bug is the most obvoius when line is at 45 degrees to meridian

**Sandcastle example**: 
https://sandcastle.cesium.com/index.html?src=Corridor.html&label=Geometries
Set position for red corridor to [-100.0, 40.0, -105.0, 35.0]

**Not working Browsers**:
 Chrome 79
 Firefox 72
Working Browsers:
IE 11

**Operating System**:
Windows 10

**Comment by strbam** [comment]
I found out that if **height** property is set then this is not happening.

**Comment by OmarShehata** [comment]
@strbam can you provide a screenshot of the issue?

**Comment by strbam** [comment]
Of course. Instead of rounded corners there are spikes for red corridor.
![cesium](https://user-images.githubusercontent.com/53467212/75264113-379c1e80-57ef-11ea-874d-3a6ecc4f8dd0.png)

**Comment by strbam** [comment]
Add screenshot of original sandcastle example for comparison
![cesium](https://user-images.githubusercontent.com/53467212/75264461-b8f3b100-57ef-11ea-858a-d93a041359a6.png)

**Comment by anne-gropler** [comment]
I ran into this issue today, too. It makes the corridor unusable for rounded corners.

In case more information is needed:
* The corner will also cut sharp edges. See [this sandcastle](https://sandcastle.cesium.com/#c=1VRRa9swEP4rh59cyGRnXR7iuWElDWOQ0dGFjTLvQbUvjZisM5Kc4I7898l2nXlpUrKHPtQPRnfffSfpuw+lpIyFtcANargAhRuYohFlzr41OT/x0iaekrJcKNSJd/Y+UYlKG6bGbEpai4xqetuHobLCCjSMZ5n/O1EAiucYQeLdYAZpV08KTKmXPEXYCLsCTaXK2gK3j0m8QU3tyiNoOgEUZFx3t3vUHXXKtXUrrs7ZUlN+hfca0VxqzSv/R0sCeDMMRywcdOG7fuCwsI+F/2KjE7Hz8S74efa42IjMriJ4G9bfDs65RS24/HsFkqTZzeyK1VJcymLF/ZCN2i5b99/uqf6FO8WeUbwgWUk3r51sqeR5saCPjcoRWF3i4BULOmyB7Z4yFUpJm/+x5G3DOO7KXLhRvZwrx8c1HB+XcDw6DTrdke31FlWBPU92Kfb502LmzPm8e29n8/n190MGhoMubmf1MkZ+JUN44uJHGR6I8gX5vde1hr2BFxtbSZx0rT+IvCBtodTSZyywmBfSDccEd2X6Cy1LjamJdWkc9KlxJtYgsosDz3stsDEOWZZSfhUPmHiTOHD1T6iSeCbU/fUateRVXbYaTuZtkjEWBy48zLRE8o7rvc5/AA) or [this sandcastle](https://sandcastle.cesium.com/#c=lVVbb9MwFP4rVnlJpNRNuxS20k1AQewBiWmr4IHy4CZuYs2xo2Nno0z97xzHyZrSjUuqJM65+PvOzU21MpbcCX7PgZwTxe/JghtRl/RLIwtWg7T5XmhlmVAcVoOIPKwUIZYDoGTWOSz9N92ALr9qkFkrCMKIjEZkPhwOCfBS33FiC2GIxN2I1SQteHqLIk6+frw5TXAF3BRaZiu1C187JHenDdGUK/s70QUDiyumToIkicev4hOanI3jZPLqdByRyWQax1OaTOIxquLJWRKRZHo2TZKXdHxydjpFyR5lj7SRWsOVFsoahPvmgb+j4cq6nzP1b29eOcsxWh6RoizLArdv7/K7Rd0W5Jlw4oicxPiIw7+a9izCP5Oc/IFkh/GP/Dy5/+TX0GvXvSTTqjZF4NMYtUx9XXpFyYFztQTcMZcc4/B9S5GtsIKbx1T3OHxwum3w4CtQabnNNfbsQ1eRVDJjxEakzAqtltuKPzb04khF331eXjaRONdCcGCQFtvZYdBSrll6ewW64oDQQUjOLxygjxm4rUH1Pa48qctuu6CXFl9Mh7aLyIZJw8NH/JJhkQSTe8Jaamjm7zHzSfBUPZIgpli3sXvE/vEy3G+MvK+0ES7qSy7yws48dGuwi3zydmGz+H1GTVVw4IvDNtM5sKoQ6SG7wDfaU/MHPFtoAJFpeKbQbQkVK7Fkq8E1z9C19dCKmBo2LOXkXtiCgK5V5g3wCDN4hnnnzqHXEFUbuZn12zPq1Pcis8WMTFzaOtkzhbj+8J469LeyKhhmfBp2iWvevcOtDe+n1uVSBwdd7sdlEA3mxm4lv+gw34iy0mBJDTKgdGR5WUmkYUbrOr3llqbGtOOD1wurNTYl7KMkxPVo3uRlRiBfsyCZ4MnY3jE93XtjTjDfQuUzklQ/euK1hozDEFgmanOo3B1BC1XVtk/gDqcDZ0sOmRQ5jmQpskzyY9Sh1RUm/AC5U621tbo81LbQ81E/YfNM3BGRnT/xd+aPANRsailvxE++GlzMR2h/5Co1c6ifkblkW2dWjC8+eSGldD7Cz73nLw).
![grafik](https://github.com/CesiumGS/cesium/assets/5690856/299fb3e0-1025-4975-a2e1-5f21ae1716b4) 
![grafik](https://github.com/CesiumGS/cesium/assets/5690856/d02dcefd-8cb5-44b4-8f0f-445895361cd0)

* The corner is perfectly round when it is showing towards north, east, west or south. But it is sharp as a triangle when heading towards north-east, north-west, south-west, south-east. See the giv, where north is up (gif is from my local app). ![roundcorner](https://github.com/CesiumGS/cesium/assets/5690856/1cf072b5-cb70-45df-ab68-9fc0813db342)
